crates/bevy_asset/src/assets.rs

```diff
@@ -133,12 +133,12 @@ impl<T: Asset> Assets<T> {
     /// This is the main method for accessing asset data from an [Assets] collection. If you need
     /// mutable access to the asset, use [`get_mut`](Assets::get_mut).
     pub fn get(&self, handle: &Handle<T>) -> Option<&T> {
-        self.assets.get(&handle.into())
+        self.assets.get::<HandleId>(&handle.into())
```

Comment:
This seems like a usability regression. What required the use of turbofishes here?

Reply:
hashbrown changed the signature of `get` from 
```rust
pub fn get<Q: ?Sized>(&self, k: &Q) -> Option<&V>
where
    K: Borrow<Q>,
    Q: Hash + Eq, 
```
https://docs.rs/hashbrown/0.12.3/hashbrown/struct.HashMap.html#method.get

to
```rust
pub fn get<Q>(&self, k: &Q) -> Option<&V>
where
    Q: Hash + Equivalent<K> + ?Sized,
```
https://docs.rs/hashbrown/0.13.2/hashbrown/struct.HashMap.html#method.get

and that seems to be less friendly to type inference

Reply:
Argh, that definitely seems like something that wouldn't fly in the stdlib. Unfortunate.